toolkit/tools/internal/customizationmacros/customizationmacros.go

```diff
@@ -0,0 +1,112 @@
+// Copyright Microsoft Corporation.
+// Licensed under the MIT License.
+
+package customizationmacros
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/file"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/logger"
+	"github.com/microsoft/azurelinux/toolkit/tools/internal/rpm"
+)
+
+const (
+	// macro files to customize the installer and final images
```

Comment:
I think we shouldn't mention the installer or final image here. This is a helper package, which shouldn't really need to have any understand the installer vs final image concepts.

Reply:
See above, reworkign comments